.github/workflows/build.yml

```diff
@@ -30,6 +30,9 @@ jobs:
       - name: Install node dependencies
         run: |
           npm ci
+      - name: Check formatting
+        run: |
+          npx prettier -u -c '*.ts' '*.js' config src static tools
```

Comment:
Is this treating .js/.ts differently to .html/.css in the GitHub workflow?

Reply:
No, they're just globs for the files that should be formatted.